sdk/tests/crosses-testnet.test.ts

```diff
@@ -43,20 +43,20 @@ describe('testnet-crosses-limitations', async () => {
     const liquidityDelta = 10000000n * 10n ** 6n
     const spotSqrtPrice = 1000000000000000000000000n
     const slippageTolerance = 0n
-    for (let i = -2560n; i < 20; i += 10n) {
+    for (let i = -1n; i < 8; i++) {
       await invariant.createPosition(
         deployer,
         poolKey,
-        i,
-        i + 10n,
+        -i * 127n - 127n,
+        -i * 127n,
         liquidityDelta,
         spotSqrtPrice,
         slippageTolerance
       )
     }
 
     const swapper = keyring.addFromUri('//Bob')
-    const swapAmount = 909000n
+    const swapAmount = 418000n
```

Comment:
It's difficult to test with a constant amount, because each swap shifts the price, and the amount needed to cross changes. Ideally, this test would retrieve the state of ticks and calculate the amount needed to cross based on that - it's quite simple since the same logic is already present in the swap